.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ permissions:
 
 jobs:
   goreleaser:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
`gon` only runs on macOS, so build everything there

Reply:
alternatively we could build on ubuntu (linux and windows) and mac (macOS), and tell goreleaser only to sign on mac, so that we use more of the faster ubuntu machines. Unfortunately I couldn't figure out configuration for this path timely.

Reply:
Let's keep it simple - we are running this workflow for releases only, which shouldn't be too frequent.